python/examples/ring.py

```diff
@@ -0,0 +1,52 @@
+# Calculating 2d ring-resonator modes, from the Meep tutorial.
+from __future__ import division
+
+import meep as mp
+
+
+def main():
+    n = 3.4  # index of waveguide
+    w = 1  # width of waveguide
+    r = 1  # inner radius of ring
+    pad = 4  # padding between waveguide and edge of PML
+    dpml = 2  # thickness of PML
+    sxy = 2 * (r + w + pad + dpml)  # cell size
+
+    # Create a ring waveguide by two overlapping cylinders - later objects
+    # take precedence over earlier objects, so we put the outer cylinder first.
+    # and the inner (air) cylinder second.
+    dielectric = mp.Medium(epsilon=n * n)
+    air = mp.Medium()
+
+    c1 = mp.Cylinder(r + w, material=dielectric)
```

Comment:
Why not `radius=r+w` rather than just `r+w` since there is also a `height` attribute which needs to be disambiguated.

Reply:
Radius is the only positional argument since there's no default, so it will always be first.  I added `radius=` for clarity though.